node/pkg/websocketfetcher/providers/gopax/gopax.go

```diff
@@ -0,0 +1,118 @@
+package gopax
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"bisonai.com/orakl/node/pkg/websocketfetcher/common"
+	"bisonai.com/orakl/node/pkg/wss"
+	"github.com/rs/zerolog/log"
+	"nhooyr.io/websocket"
+)
+
+type GopaxFetcher common.Fetcher
+
+func New(ctx context.Context, opts ...common.FetcherOption) (common.FetcherInterface, error) {
+	config := &common.FetcherConfig{}
+	for _, opt := range opts {
+		opt(config)
+	}
+
+	fetcher := &GopaxFetcher{}
+	fetcher.FeedMap = config.FeedMaps.Separated
+	fetcher.FeedDataBuffer = config.FeedDataBuffer
+
+	subscription := []any{
+		Subscription{
+			Name: "SubscribeToTickers",
+		},
+	}
+
+	ws, err := wss.NewWebsocketHelper(ctx,
+		wss.WithEndpoint(URL),
+		wss.WithSubscriptions(subscription),
+		wss.WithProxyUrl(config.Proxy),
+		wss.WithReadLimit(32769*10),
```

Comment:
how about declaring these numbers as constants and leaving a small explanation (including the unit in the const name is also an option)